example/src/main/kotlin/com.expedia.graphql.sample/directives/StringEvalDirectiveWiring.kt

```diff
@@ -0,0 +1,47 @@
+package com.expedia.graphql.sample.directives
+
+import com.expedia.graphql.sample.directives.DirectiveWiring.Companion.getDirectiveName
+import graphql.schema.DataFetcher
+import graphql.schema.DataFetchingEnvironmentBuilder
+import graphql.schema.GraphQLFieldDefinition
+import graphql.schema.idl.SchemaDirectiveWiringEnvironment
+
+class StringEvalDirectiveWiring : DirectiveWiring {
+    private val dirName = getDirectiveName(StringEval::class)
+
+    override fun isResponsible(environment: SchemaDirectiveWiringEnvironment<*>): Boolean {
+        (environment.element as? GraphQLFieldDefinition)?.arguments?.forEach { arg ->
```

Comment:
instead of `forEach` i think you could use `any( predicate )`

Reply:
:+1: